{"name": "test_checklogin[test_login2]", "status": "broken", "statusDetails": {"message": "KeyError: 'err_code'", "trace": "test_login = ('http://hn216.api.yesapi.cn/', 'App.User.Login', '', '', '784DBD349CE644ABD3684A562C159585')\n\n    def test_checklogin(test_login):\n    \n        checkloginusr=src.test.common.checklogin.CheckLogin()\n        checkinfo=checkloginusr.check_login(test_login[0],\"App.User.Check\",test_login[2],test_login[3],test_login[4])\n        # checkinfo=checkloginusr.check_login(url,service,uuid,token,app_key)\n        print(\"test_login[0]======\",test_login[0])\n        print(\"test_login[1]======\",test_login[1])\n        print(\"test_login[2]======\",test_login[2])\n        print(\"test_login[3]======\",test_login[3])\n        print(\"test_login[4]======\",test_login[4])\n    \n        x=json.loads(checkinfo.text)\n        print(\"x===========\",x)\n    \n    \n>       err_code=x[\"data\"][\"err_code\"]\nE       KeyError: 'err_code'\n\ntest_checklogin3.py:41: KeyError"}, "parameters": [{"name": "test_login", "value": "['http://hn216.api.yesapi.cn/', 'App.User.Login', 'autotest401', '174b086fc6358db6154bd951a8947837', 1.0, '784DBD349CE644ABD3684A562C159585']"}], "start": 1572332345478, "stop": 1572332345570, "uuid": "301826dc-6d19-46c1-9a31-dc0b36e9907b", "historyId": "2b089518a18ecade6e724fe4f3e24fc2", "testCaseId": "0060407f2fc5ffcce8df6c8eb6c4f5f3", "fullName": "src.test.case.test_checklogin3#test_checklogin", "labels": [{"name": "parentSuite", "value": "src.test.case"}, {"name": "suite", "value": "test_checklogin3"}, {"name": "host", "value": "DESKTOP-4A0JR1L"}, {"name": "thread", "value": "4936-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.test.case.test_checklogin3"}]}